cmd/create/accountroles/validation.go

```diff
@@ -0,0 +1,36 @@
+package accountroles
+
+import errors "github.com/zgalor/weberr"
+
+func validateSharedVpcInputs(hostedCp bool, vpcEndpointRoleArn string,
+	route53RoleArn string) (bool, error) {
+
+	if !hostedCp {
+		if vpcEndpointRoleArn != "" {
```

Comment:
@hunterkepley Take a look at the `machinepools.HostedClusterOnlyFlag` function. Is there anything we can do to at least standardise the messaging we give to a user when they're trying to user a HCP-only flag with a classic cluster?